operator/src/main/resources/scripts/readState.sh

```diff
@@ -26,7 +26,7 @@ if [ `jps -v | grep -c " -Dweblogic.Name=${SERVER_NAME} "` -eq 0 ]; then
 fi
 
 if [ ! -f ${STATEFILE} ]; then
-  trace "Error: WebLogic Server state file not found."
+  trace "WebLogic Server state file not found."
```

Comment:
Should this be SEVERE?

Reply:
I don't think so.  I removed it since this file is called by stopServer.sh, which assumes things are OK when readState.sh itself returns non-zero.

Reply:
I think readState.sh is also called from ServerStatusReader.java

Reply:
I just checked ServerStatusReader.java, and it assumes that it's normal to hit this path:
```
               proc = kubernetesExec.exec("/weblogic-operator/scripts/readState.sh");
               ...
               int exitValue = proc.exitValue();
               ...
                } else if (exitValue == 1 || exitValue == 2) {
                  state =
                      PodHelper.isDeleting(pod)
                          ? WebLogicConstants.SHUTDOWN_STATE
                          : WebLogicConstants.STARTING_STATE;
                } else {
```

Reply:
So it looks like this path also doesn't merit an 'error' or 'severe' at line 29.

Reply:
Yes, if we read the status very early in the life of the pod.